src/en/controllers-backup.md

```diff
@@ -1,385 +1,278 @@
-size (B):        45565185
-stored:          2016-04-29 09:20:59 +0000 UTC
-started:         2016-04-29 09:20:34 +0000 UTC
-finished:        2016-04-29 09:20:50 +0000 UTC
-notes:           ""
-model ID:        "e94566bc-d02d-4a14-8ec2-e2dbed2f2ec4"
+size (B):        58605244
+stored:          2018-05-15 19:40:28 +0000 UTC
+started:         2018-05-15 19:37:24 +0000 UTC
+finished:        2018-05-15 19:37:41 +0000 UTC
+notes:           "fresh lxd controller"
+model ID:        "9c6a3650-2957-489a-8f0c-6c3b5ce2e055"
 machine ID:      "0"
-created on host: "juju-e94566bc-d02d-4a14-8ec2-e2dbed2f2ec4-machine-0"
-juju version:    2.0
+created on host: "juju-e2e055-0"
+juju version:    2.3.7
 ```
 
-### juju remove-backup  
-usage: `juju remove-backup [-m | --model] <ID>`
+The `started` time is the most pertinent of the various timestamps. It refers
+to the time the backup was created.
 
-If you wish to remove a particular backup file from the controller (perhaps 
-to save space!), you can use the `remove-backup` command with the appropriate 
-ID:
+#### `juju download-backup`
 
-```bash 
-juju remove-backup 20160429-092034.e94566bc-d02d-4a14-8ec2-e2dbed2f2ec4
-```
-After a short time, you should see a confirmation:
+The `download-backup` command downloads a specific remote backup (again,
+identified via the `backups` command). Here, we download a backup that is
+stored on the 'aws' controller:
 
 ```bash
-successfully removed: 20160429-092034.e94566bc-d02d-4a14-8ec2-e2dbed2f2ec4
+juju download-backup -m aws:controller 20180515-191942.7e45250b-637a-4dc9-8389-c6aa70100cd6
 ```
 
+#### `juju upload-backup`
 
-### juju upload-backup  
-usage: `juju upload-backup [-m | --model] <filename>`
-
-As well as downloading backups from the controller, it is also possible
-to upload them. This can be useful either to break up the process of restoring 
-from a backup (upload the file, then restore using the ID), or in the case 
-where backups have been removed from the state server. On completion, the 
-command will return all the metadata for the uploaded backup file. 
-
-Examples:
+The `upload-backup` command uploads a specific local backup to a controller.
+For example:
 
 ```bash
-juju upload-backup backup.tar.gz
+juju upload-backup -m lxd:controller juju-backup-20180515-193724.tar.gz
 ```
-will result in output such as:
 
-```no-highlight
-backup ID:       "20160429-092034.e94566bc-d02d-4a14-8ec2-e2dbed2f2ec4"
-checksum:        "7ChuK/4IWCzd4XysP9j0UMFRCL8="
-checksum format: "SHA-1, base64 encoded"
-size (B):        45565185
-stored:          2016-04-29 10:46:22 +0000 UTC
-started:         2016-04-29 09:20:34 +0000 UTC
-finished:        2016-04-29 10:32:35 +0000 UTC
-notes:           ""
-model ID:        "e94566bc-d02d-4a14-8ec2-e2dbed2f2ec4"
-machine ID:      "0"
-created on host: "juju-e94566bc-d02d-4a14-8ec2-e2dbed2f2ec4-machine-0"
-juju version:    2.0
-```
+!!! Note:
+    It is not possible to upload a file that corresponds to a backup stored
```

Comment:
not sure I understand this. Do you mean with the same name?

Reply:
No, the same backup. I'll reword then.